core/src/main/java/com/linecorp/armeria/common/AbstractRequestContextBuilder.java

```diff
@@ -98,7 +100,7 @@ protected AbstractRequestContextBuilder(boolean server, HttpRequest req) {
         this.req = requireNonNull(req, "req");
         rpcReq = null;
         sessionProtocol = SessionProtocol.H2C;
-
+        uuid = UUID.randomUUID();
```

Comment:
Could you begin `uuid` with `null`, and call `UUID.randomUUID()` only when a user did not set the UUID of the context?